pkg/network/deviceinfo/deviceinfo_suite_test.go

```diff
@@ -22,9 +22,45 @@ package deviceinfo_test
 import (
 	"testing"
 
+	virtv1 "kubevirt.io/api/core/v1"
+
 	"kubevirt.io/client-go/testutils"
 )
 
 func TestDeviceInfo(t *testing.T) {
 	testutils.KubeVirtTestSuiteSetup(t)
 }
+
+func newBridgeInterface(name string) virtv1.Interface {
```

Comment:
Can you use `libvmi` for this one and all the rest?

Reply:
done, much nicer:)